common.mk

```diff
@@ -79,9 +82,17 @@ tag: build
 
 .PHONY: clean
 clean:
+	test -f auto_targets.mk && rm auto_targets.mk
```

Comment:
@dhodovsk Moving this line directly into the `clean.sh` script should get rid of the test failures.

Reply:
yeas, sorry about that :)